sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1090,8 +1090,8 @@ public partial class JsonObjectSerializer : Azure.Core.Serialization.ObjectSeria
     }
     public static partial class ModelSerializer
     {
-        public static T Deserialize<T>(System.IO.Stream stream, Azure.Core.Serialization.SerializableOptions? options = null) where T : Azure.Core.Serialization.IJsonSerializable, new() { throw null; }
-        public static T Deserialize<T>(string json, Azure.Core.Serialization.SerializableOptions? options = null) where T : Azure.Core.Serialization.IJsonSerializable, new() { throw null; }
+        public static T? Deserialize<T>(System.IO.Stream stream, Azure.Core.Serialization.SerializableOptions? options = null) where T : Azure.Core.Serialization.IJsonSerializable, new() { throw null; }
```

Comment:
I have thought about this more, lets throw if objectserializer gives us null, just use invalidoperationexception for now we can figure out those details later that way this signature can remain `T` without the `?`